qadence/circuit.py

```diff
@@ -7,8 +7,9 @@
 
 from sympy import Array, Basic
 
-from qadence.blocks import AbstractBlock, AnalogBlock, CompositeBlock, chain
+from qadence.blocks import AbstractBlock, AnalogBlock, CompositeBlock
 from qadence.blocks.utils import parameters, primitive_blocks
+from qadence.operations import chain
```

Comment:
This import does not look correct. Before it was fine.

Reply:
Fixed.